modules/functional-test/src/main/resources/config.properties

```diff
@@ -39,6 +39,7 @@ api-server-protocol=http
 api-version-string=v1
 user-name=admin
 password=admin
+test-user=wasabi_reader@example.com
```

Comment:
@sudheerkethireddy is it possible to reuse the existing user for this test case? this will avoid adding this test-user to the config.properties of <company-specific>-functional-test module.